giraffe/src/components/Geo.tsx

```diff
@@ -3,7 +3,6 @@ import React, {FunctionComponent, useEffect, useState} from 'react'
 import BingLayer from './geo/bing-maps/Bing'
 import {Map, TileLayer, LayersControl} from 'react-leaflet'
 import Control from 'react-leaflet-control'
-import 'leaflet/dist/leaflet.css'
```

Comment:
so if i understand things correctly, importing this file made everything work, and this PR is all the code needed to remove this import?

Reply:
Yes - if you have an application which is using giraffe now you have to explicitly import leaflet.css as you mentioned. This is an implementation detail and should be hidden from the user.